pkg/controllers/gitrepository/controller_test.go

```diff
@@ -7,10 +7,11 @@ import (
 	"os"
 	"path/filepath"
 	"reflect"
-	ctrl "sigs.k8s.io/controller-runtime"
 	"testing"
 	"time"
 
+	ctrl "sigs.k8s.io/controller-runtime"
+
```

Comment:
necessary?

Reply:
Different go import sorting. We aren't being consistent at all right now but we should standardize on goimports